stbt_rig.py

```diff
@@ -520,8 +531,9 @@ def find_test_pack_root():
         if os.path.exists(os.path.join(root, '.stbt.conf')):
             return root
         root = os.path.split(root)[0]
-    raise Exception("""Didn't find "stbt.conf" at the root of your test-pack """
-                    """(starting at %s)""" % os.getcwd())
+    raise NotInTestPack(
+        """Didn't find "stbt.conf" at the root of your test-pack """
```

Comment:
Would be worth fixing this typo, it's `".stbt.conf"` with a leading `.`.

Reply:
Fixed